packs/stormforge-agent-2.0.3/schema.yaml

```diff
@@ -0,0 +1,7 @@
+charts.stormforge-agent.authorization.clientID:
+  schema: '{{ required }}'
```

Comment:
A hint will be useful in all these schema for novice users. Check [doc](https://docs.spectrocloud.com/registries-and-packs/pack-constraints?Pack+Constraints+Format+Types=pack_constraint_format_hints#schema-constraints) on hints usage. Will give a tooltip in the UI for users to understand what these fields are.